Shared/Extensions/URLExtensions.swift

```diff
@@ -361,7 +361,14 @@ extension URL {
 extension URL {
     public var isErrorPageURL: Bool {
         if let host = self.host {
-            return self.scheme == "http" && host == "localhost" && path == "/errors/error.html"
+            return self.scheme == "http" && host == "localhost" && path.contains("/errors/")
```

Comment:
Can't this entire function just be:
`return `return self.scheme == "http" && self.host == "localhost" && path.contains("/errors/")`

Since this would account for `host` being `nil` (as in not equal to "localhost")?